test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -21,6 +24,15 @@
   ([resp keywordize?]
    (-> resp :body slurp (json/parse-string keywordize?))))
 
+(defn jolokia-encode
```

Comment:
@camlow325 Thanks for the reminder! Just filed [DOC-3051](https://tickets.puppetlabs.com/browse/DOC-3051) for this.